data-studio/data-studio.md

```diff
@@ -0,0 +1,105 @@
+## Create reports and dashboards with Data Studio
+
+![](images/example_dashboard.png)
+
+This is a short tutorial on how to set up a dashboard (or report as stated by
+data studio) in Google Data Studio (this can easily be done in other tools such
+as [Tableu](https://www.tableau.com/)).
+
+The aim is to show the potential of what you can do with the data from our
+Wittra sensor network.
+
+### Prerequisits
+
+In order to produce the dashboard it assumes you have already set up a
+database, to store the streamed data from the network. In this example we use
+[BigQuery](https://cloud.google.com/bigquery/what-is-bigquery), an enerprise
+data warehouse by Google.
```

Comment:
Setting up the database is a pretty big step. We could point the users in the direction of Google provided solutions, such as [this one](https://cloud.google.com/dataflow/docs/guides/templates/provided-streaming#running-the-pubsub-topic-to-bigquery-template).